trust the exit status of the program

Patch by Robin Getz <rgetz@blackfin.uclinux.org>

--- a/dhry_1.c
+++ b/dhry_1.c
@@ -274,6 +289,7 @@
     printf ("\n");
   }
   
+  exit(0);
 }
 
 
